content/blog/2019/12/2019-12-12-board-election-results.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "2019 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- oleg_nenashev
+opengraph:
+  image: /images/post-images/elections-2019/election-2019-opengraph.png
+---
+
+image:/images/post-images/elections-2019/election-2019-opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community is conducting the 2019 elections for Board and Officer positions.
+The call for nominations has concluded on Nov 25, and the election results have been link:https://groups.google.com/forum/#!topic/jenkinsci-dev/bX2qhzoEWXo[announced in the developer mailing list] on November 28.
```

Comment:
"The call for nominations concluded on Nov 25 and the election results were link:https://groups.google.com/forum/#!topic/jenkinsci-dev/bX2qhzoEWXo[announced in the developer mailing list] on Nov 28"

Use "Nov" here since it was used on the previous line, or change both to "November".